docs/getting-started-guides/rkt/index.md

```diff
@@ -10,25 +10,30 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
   *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
 
 - Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
-  The minimum version required for now is [v0.8.0](https://github.com/coreos/rkt/releases/tag/v0.8.0).
+  The minimum version required is [v1.9.0](https://github.com/coreos/rkt/releases/tag/v1.9.0).
```

Comment:
Line 1:5 can be deleted I think.

Line 9: replace `at this moment (2015/09/01)` to Kubernetes 1.3 ?